web/app/scripts/filterbar/filterbar.html

```diff
@@ -13,7 +13,7 @@
             <li text-search-field="__createdBy"></li>
         </ul>
         <ul class="nav navbar-nav" ng-repeat="(key, value) in filterbar.filterables">
-            <li ng-if="value.format === 'number'"
+            <li ng-if="(value.fieldType === 'integer') || (value.fieldType === 'number')"
```

Comment:
I don't remember why this is the case, but there are two routes to defining a number field: you can define a text field with the "number" format, or you can define a number field. I think this may be the behavior of the form editor library we're using. I suspect that `value.format === 'number'` detects the first case, so we should probably leave it in the condition unless / until we change or update the form builder to only have one way to define a numeric field.

Reply:
Good call, I forgot about those. Gave it a try and it looks like you're right, text number fields come out with `"format": "number"` but `"fieldType": "text"`
```json
{  
   "driverIncidentDetails#Wind speed":{  
      "isSearchable":true,
      "propertyOrder":7,
      "type":"string",
      "fieldType":"text",
      "format":"number",
      "multiple":false,
      "propertyName":"Wind speed"
   }
}
```
Added that back in!